### ISSUE TITLE
What's your training strategy?

### ISSUE THREAD

**Issue by mudcube** [ixartz/handwritten-digit-recognition-tensorflowjs#6]
Very impressive results! It's so good it almost reads my mind.

I've tried retraining the model with the settings in the provided demo many many times now, and have yet to come close to the accuracy of the pre-trained model. 

It would be really interesting to hear your training strategy in achieving 0.997 accuracy.

For example, if you could recommend what batch size, iterations, or any other tips/tricks you used to get those results? Also, did you end up using JS to train, or another language, or outside model?

**Comment by ixartz** [comment]
The pre-trained model is generated from outside using classic Tensorflow (without .js) and with GPU. So, what is why the model is more accurate with 0.997 accuracy.

After trained, we load the model in the demo.

You can also trained the JS on your mobile and I do not want to burn all the battery of the users. That explained why the trained model in JS is not optimized to achieve good results.

**Comment by mudcube** [comment]
Thanks for the details, that helps a lot :)
The following article seems useful for anyone else interested in training MNIST for high accuracy: 
https://www.kaggle.com/cdeotte/25-million-images-0-99757-mnist
